direct.go

```diff
@@ -87,25 +95,6 @@ func (d *direct) providerFor(m *masquerade) *Provider {
 	return d.providers[pid]
 }
 
-// Vet vets the specified Masquerade, verifying certificate using the given CertPool
```

Comment:
Moved to the bottom of the file. It doesn't seem to be used anywhere except in the tests. Do you by chance remember why it was introduced @oxtoacart ?

Reply:
Sorry, I don't remember. I don't see it being used anywhere either (I checked flashlight and checkfallbacks). I think it's okay to remove if you want.

Reply:
Thanks. Let me just leave it at the bottom.